### DISCUSSION TITLE
Just wondering if there is a "I gave feedback and all I got is this lousy Gave Feedback-badge"

### DISCUSSION THREAD

**Discussion by MichielBier** [community/community#28035] [Other Feature Feedback, Questions, & Ideas]
Just wondering if there is a "I gave feedback and all I got is this lousy Gave Feedback-badge", if there isn't there should be one. :D

**Comment by jfullstackdev** [comment]
what would be the benefit of that badge ?

**Comment by baraaj** [comment]
How can get this badge?
